src/build/env.py

```diff
@@ -183,7 +183,7 @@ def executable(self) -> str:
     def scripts_dir(self) -> str:
         return self._scripts_dir
 
-    def install(self, requirements: Iterable[str]) -> None:
+    def install(self, requirements: Collection[str]) -> None:
```

Comment:
FYI, this is checked just below (`if not requirements`), which is invalid on an `Iterable` - it just happens to be true for some common iterables like set, list, and tuple. Caught by `"truthy-bool"`. I've changed it to `Collection`.